src/index.html

```diff
@@ -22,6 +22,359 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo of Moyo"
+        />
+      </a>
+      <nav class="navbar">
+        <ul class="navbar__ul">
+          <li class="navbar__li">
+            <a
+              href="#"
+              class="is-active navbar__link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="navbar__li">
+            <a
+              class="navbar__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="navbar__li">
+            <a
+              class="navbar__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="navbar__li">
+            <a
+              class="navbar__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="navbar__li">
+            <a
+              class="navbar__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="navbar__li">
+            <a
+              class="navbar__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="navbar__li">
+            <a
+              class="navbar__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="navbar__li">
+            <a
+              class="navbar__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        />
+
+        <h2 class="card__text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <span class="card__product-code">Product code: 195434</span>
```

Comment:
use the semantic tag where possible(in all the cases)

Reply:
> use the semantic tag where possible(in all the cases)

Not fixed